Packs/CiscoEmailSecurity/Integrations/CiscoEmailSecurity/CiscoEmailSecurity.yml

```diff
@@ -25,8 +25,8 @@ configuration:
   name: timeout
   required: false
   type: 0
-description: Cisco Email Security is an email security gateway . It detects and blocks a wide variety of email-borne threats, such as malware, spam and phishing.
-display: CiscoEmailSecurity (Beta)
+description: Deprecated. Use CiscoSMA instead.
```

Comment:
@jonathantauman Please change to "Deprecated. Use Cisco Security Management Appliance instead."